static/i18n/en/app.json

```diff
@@ -401,12 +401,20 @@
     },
     "firmware": {
       "installed": "Firmware version {{version}}",
-      "update": "Update",
+      "updateLater": "Update later",
+      "seedReady": "I have my recovery phrase",
+      "dontHaveSeed": "Don't have your recovery phrase ?",
+      "followTheGuide": "Follow our step by step update guide",
+      "removeApps": "Uninstall all apps before updating",
+      "update": "Firmware update",
       "latest": "Firmware version {{version}} is available",
+      "prepareSeed": "Make sure your 24-word recovery phrase is written down on your recovery sheet and accessible.",
       "disclaimerTitle": "You are about to install <1><0>firmware version {{version}}</0></1>.",
       "disclaimerAppDelete": "Uninstall all apps from your device. This does not affect your crypto assets.",
       "disclaimerAppReinstall": "Ensure your Recovery sheet is accessible, as a precaution.",
-      "downloadingUpdateDesc": "Please wait while the update installer is being downloaded"
+      "downloadingUpdateDesc": "Please wait while the update installer is being downloaded",
+      "appsAutoUninstallTitle": "Apps will be uninstalled",
+      "appsAutoUninstallDesc": "In order to update device’s firmware, all applications on your device will be uninstalled. You will be able to reinstall them once the update is complete."
```

Comment:
All applications will be uninstalled from your device for the firmware update. You will be able to reinstall them after the update. Don't worry, this does not affect your crypto assets.